src/include/migraphx/module.hpp

```diff
@@ -2,11 +2,12 @@
 #define MIGRAPHX_GUARD_MIGRAPHLIB_MODULE_HPP
 
 #include <list>
+#include <unordered_set>
 #include <unordered_map>
 #include <migraphx/operation.hpp>
 #include <migraphx/literal.hpp>
 #include <migraphx/builtin.hpp>
-#include <migraphx/instruction_ref.hpp>
+#include <migraphx/instruction.hpp>
```

Comment:
This shouldn't include the instruction header here.

Reply:
if just including "instruction_ref.hpp", there will be a build error for the 
std::unordered_map<instruction_ref, std::string> map_name = {}.

That is why i put the default argument value in the .cpp file last time.

Reply:
I see, then we should add two overloads for this function. The one that doesnt have the names parameter can call the other overload and create the empty map.

Reply:
fixed.